openreview/venue_request/venue_request.py

```diff
@@ -128,6 +128,14 @@ def setup_bidding_stage(self):
     def setup_review_stage(self):
 
         review_stage_content = {
+            'review_name': {
+                'description': 'What should be the name of the official review button? Use underscores to represent spaces. Default name: Official Review',
```

Comment:
If we are asking for underscores instead of spaces, should this be:

> Default name: Official_Review'

Reply:
I think you're right, I updated the description